docs/examples/clusters/glusterfs/userdata/bootstrap

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
```

Comment:
General comments:
1. Indentation looks off so not easy to read.
1. I would break into at least two files such as bootstrap_server and bootstrap_client.
1. I would try to avoid duplicated code by parameterizing the common logic by introducing local vars which are set based on the similar case statement.
1. Just curious, the device and dir names "brick[123]" need to be numbered?  Can't it be the same name among the three servers?

Reply:
1. Good catch, need to fix the indentation settings in my IDE :)
2. I also considered this. Sounds like it would be a good idea to separate the server/client logic.
3. This is a good point. I will fix this.
4. There's no reason to have them be numbered. Will fix this.